tests/e2e/specs/out_of_game/home.spec.js

```diff
@@ -18,6 +18,18 @@ function assertSuccessfulJoin(gameState) {
   cy.contains('h1', `Lobby for ${gameState.name}`);
 }
 
+describe('Vue Devtools', () => {
```

Comment:
I added an issue for this, https://github.com/cuttle-cards/cuttle/issues/41. I think that there needs to be a standard around when e2e tests are run. It feels like only against a prod build makes sense so we can properly assert here that the dev script doesn't exist.

That said, as of today you can run either a dev build OR a prod build and run e2e tests against either. I know you use it as a way to jump in to a given scenario so maybe there's a middleground here I'm not thinking of?

Reply:
I think you're right and this is speaking to the value of programatic environment-specific test configuration which runs different tests depending on the environment. This is tricky stuff we'll want to think hard on and I'm really excited to dig into it together